.github/workflows/nexus-feature-branch.yml

```diff
@@ -4,6 +4,12 @@ on:
   push:
     branches:
       - CP-*
```

Comment:
```suggestion
```
Hey, does this change meet our needs? I think a push to CP-* will still trigger the workflow, ignoring the pull_request conditions. Maybe we should remove push and use something like:

```
if: startsWith(github.ref, 'refs/heads/CP-')
```
What do you think? 😊